crypto/data_formats_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package crypto
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/google/trillian"
+)
+
+const (
+	publicKeyFile      = "../testdata/log-rpc-server.pubkey.pem"
+	privateKeyFile     = "../testdata/log-rpc-server.privkey.pem"
```

Comment:
These paths might not work inside of Google. Instead, could you import "github.com/google/trillian/testonly" and use testonly.Demo{Private,Public}Key?